ss58-registry.json

```diff
@@ -460,11 +460,20 @@
       "standardAccount": "*25519",
       "website": "https://composable.finance"
     },
+    {
+      "prefix": 51,
+      "network": "turing",
+      "displayName": "Turing Network",
+      "symbols": ["TUR"],
+      "decimals": [10, 10],
+      "standardAccount": "*25519",
+      "website": "https://oak.tech/turing/home/"
+    },
     {
       "prefix": 51,
       "network": "oak",
       "displayName": "OAK Network",
-      "symbols": ["OAK", "TUR"],
+      "symbols": ["OAK"],
```

Comment:
Hmm so the previous value ["OAK", "TUR"] doesn’t work for subkey? How is this symbols array translated to subkey? Maybe we should just make subkey accept two values in the array.

Reply:
Yes, It doesn't work for subkey.

In fact, only the prefix field is read from the ss-58 module in the subkey code, and other fields are not read.